game/script.rpy

```diff
@@ -65,8 +65,6 @@ menu:
     "I want lots of money and stability to be dangerously honest.":
         jump badend
 
-    "I wanted a hands-on career.":
-        jump mechE
```

Comment:
Thanks for deleting this. Just as we discussed. :)

Reply:
No problem! now it should work with the new file!